packages/x-license/package.json

```diff
@@ -11,17 +11,15 @@
   "homepage": "https://mui.com/x/introduction/licensing/",
   "sideEffects": [
     "./utils/licenseInfo.js",
-    "./modern/utils/licenseInfo.js",
-    "./node/utils/licenseInfo.js"
+    "./esm/utils/licenseInfo.js"
```

Comment:
🤔  This shouldn't be necessary I believ

Reply:
this property is copied to the package.json in the esm folder, so the relative location should be the same

Reply:
On the other hand, the `package.json` on the `esm` folder looks like this:
<img width="418" alt="Screenshot 2025-04-14 at 22 09 27" src="https://github.com/user-attachments/assets/2d20da66-e6e7-410d-9a72-480f994a8428" />

Is it expected? 🤔